portfolio/src/main/webapp/gameScript.js

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+//fecth two truths and one lie from the server
+function getFacts() {
+    fetch('/two-truths-one-lie').then(response => response.json()).then((game) => {
+        facts = game.twoTruthsOneLie;
+        console.log(facts);
```

Comment:
nit: either remove this, or add some context to what it contains (like "Facts received from server: ")

Reply:
fixed